--- a/build.rs
+++ b/build.rs
@@ -76,6 +76,7 @@ fn link_opus(is_static: bool, opus_build_dir: impl Display) {
     println!(
         "cargo:info=Linking Opus as {} lib: {}",
         is_static_text, opus_build_dir
     );
     println!("cargo:rustc-link-lib={}=opus", is_static_text);
     println!("cargo:rustc-link-search=native={}/lib", opus_build_dir);
+    println!("cargo:rustc-link-search=native={}/lib64", opus_build_dir);
 }
